spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -360,3 +360,212 @@ Another option to start a task using the `task-launcher` would be to create a st
+```
+.Basic Transition With Wildcard
+image::{dataflow-asciidoc}/images/dataflow-ctr-transition-basic-wildcard.png[Composed Task Basic Transition with Wildcard, scaledwidth="50%"]
+
+In the example above `foo` would launch and if it had an exit status of `FAILED`,
+then the `bar` task would launch. Any exit status of `foo` other than `FAILED`
+then `baz` would launch.
+
+===== Transition With a Following Conditional Execution
+A transition can be followed by a conditional execution so long as the wildcard
+is not used. For example:
+```
+task create my-transition-conditional-execution-task --definition "foo 'FAILED' -> bar 'UNKNOWN' -> baz && qux && quux"
+```
+.Transition With Conditional Execution
+image::{dataflow-asciidoc}/images/dataflow-ctr-transition-conditional-execution.png[Composed Task Transition with Conditional Execution, scaledwidth="50%"]
+
+In the example above `foo` would launch and if it had an exit status of `FAILED`,
+then the `bar` task would launch.  If `foo` had an exit status of `UNKNOWN` then
+`baz` would launch.  Any exit status of `foo` other than `FAILED` or `UNKNOWN`
+then `qux` would launch and upon successful completion `quux` would launch.
+
+==== Split Execution
+Splits allow for multiple tasks within a composed task to be run in parallel.
+It is denoted by using angle brackets <> to group tasks and flows that are to
+be run in parallel. These tasks and flows are separated by the double pipe `||`
+. For example:
+```
+task create my-split-task --definition "<foo || bar || baz>"
+```
+.Split
+image::{dataflow-asciidoc}/images/dataflow-ctr-split.png[Composed Task Split, scaledwidth="50%"]
+The example above will launch tasks `foo`, `bar` and `baz` in parallel.
+With the task DSL a user may also execute multiple split groups
+in succession. For example:
+```
+task create my-split-task --definition "<foo || bar || baz> && <qux || quux>"
+```
+
+.Split as a part of a conditional execution
+image::{dataflow-asciidoc}/images/dataflow-ctr-multiple-splits.png[Composed Task Split, scaledwidth="50%"]
+In the example above tasks `foo`, `bar` and `baz` will be launched in parallel,
+once they all complete then tasks `qux`, `quux` will be launched in parallel.
+Once they complete the composed task will end.   However if `foo`, `bar`, or
+`baz` fails then, the split containing `qux` and `quux` will not launch.
+
+===== Split Containing Conditional Execution
+A split can also have a conditional execution within the angle brackets.  For
+example:
+```
+task create my-split-task --definition "<foo && bar || baz>"
+```
+.Split with conditional execution
+image::{dataflow-asciidoc}/images/dataflow-ctr-split-contains-conditional.png[Composed Task Split With Conditional Execution, scaledwidth="50%"]
+
+In the example above we see that `foo` and `baz` will be launched in parallel,
+however `bar` will not launch until `foo` completes successfully.
+
+=== Configuring Composed Task in Spring Cloud Data Flow
```

Comment:
Beforementioned examples include task executions already, but it will _only_ work if the CTR is registered and available. So, I'd suggest moving this section to the beginning.